presets/4.3/vtx/eachine_vtx.txt

```diff
@@ -22,37 +21,37 @@ vtxtable band 5 RACEBAND R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
 
 #$ OPTION_GROUP BEGIN: choose your vtx
 
-#$ OPTION BEGIN (UNCHECKED): Eachine TX805
+#$ OPTION BEGIN (UNCHECKED): Eachine TX805 (SA) 
```

Comment:
please mention which SA version is that exactly, 1.0, 2.0, 2.1 (like it is in the other vtx tables)
And always make sure you got 1 commit for the PR (squash) so that its ready to merge.